src/styles/styles/card.scss

```diff
@@ -0,0 +1,147 @@
+.catalog__content {
+  box-sizing: border-box;
+  display: grid;
+  grid-template-rows: auto;
+  grid-template-columns: 200px;
+
+  justify-content: center;
+  gap: 46px 48px;
+  padding: 50px 40px ;
+  margin: 0 auto;
+
+  @media (min-width: 488px) {
+    grid-template-columns: 200px 200px;
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: 200px 200px 200px;
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: 200px 200px 200px 200px;
+  }
+  }
+
+  .card {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 200px;
+    padding: 32px 16px 16px;
+    box-sizing: border-box;
+    border: 1px solid $border-color;
+    border-radius: 5px;
+    transition: 0.3s;
+
+    &:hover {
+      transform: scale(1.2);
+    }
+
+    &__img {
+      margin: auto;
+      width: 160px;
+      height: 134px;
+      margin-bottom: 40px;
+    }
+
+    &__title {
+      margin: 0;
+      margin-bottom: 4px;
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+      color: $main-color;
+      transition: 0.3s;
```

Comment:
write property for which you use `transition` like `color`

check all your `transition` rule all over the file and update according to comment